tests/test_docs.py

```diff
@@ -41,6 +41,7 @@ class TestDocument(unittest.TestCase):
     OUTPUT_XML_DIR_PATH = path.abspath(path.join(DIR_NAME, OUTPUT_XML_DIR))
     EXPECTED_XML_DIR_PATH = path.abspath(path.join(DIR_NAME, EXPECTED_XML_DIR))
 
+    @unittest.skip("DOC_DIR_PATH doesn't exist")
```

Comment:
Why this needs to be skipped?

Reply:
`DOC_DIR = 'doc-test-files'` doesn't exist anywhere in the repo. I even went back to a very old commit and still can't find it so I don't know what's going on here. These tests don't even fail, they just throw warnings.